src/pages/AddServers.js

```diff
@@ -222,7 +244,20 @@ class AddServers extends BaseUpdateWizardPage {
   }
 
   isInstallable = () => {
-    return this.isDeployable();
+    if(this.state.model && this.state.model.size > 0) {
```

Comment:
I don't understand why the size check is here. When would you ever have a model which is not `undefined` but which is empty?

Reply:
at certain point during testing...noticed model.size could be 0 sometimes.... Added it to void problems.